tools/prove_blocks.sh

```diff
@@ -81,7 +81,7 @@ fi
 if [[ $5 == "test_only" ]]; then
     # test only run
     echo "Proving blocks ${BLOCK_INTERVAL} in a test_only mode now... (Total: ${TOT_BLOCKS})"
-    cargo r --release --features test_only --bin leader -- --runtime in-memory --load-strategy on-demand jerigon --rpc-url "$NODE_RPC_URL" --block-interval $BLOCK_INTERVAL --proof-output-dir $PROOF_OUTPUT_DIR $PREV_PROOF_EXTRA_ARG > $OUT_LOG_PATH 2>&1
+    cargo r --release --features test_only --bin leader -- --runtime in-memory --load-strategy on-demand jerigon --rpc-url "$NODE_RPC_URL" --block-interval $BLOCK_INTERVAL --proof-output-dir $PROOF_OUTPUT_DIR $PREV_PROOF_EXTRA_ARG 2>&1 > $OUT_LOG_PATH
```

Comment:
Is it actually changing anything? I see it as still redirecting `stdout` and `stderr` to `$OUT_LOG_PATH`, similarly to calling `&> $OUT_LOG_PATH` directly.

Reply:
I was actually paranoid as I didn't saw some output. Will revert it